### PR TITLE
Fixes a bunch of typos

### PR DESCRIPTION
**Pull request by Llywelwyn**
see title

i started by intending to fix one and now here we are

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!review !bugfix

**Comment by Llywelwyn** [comment]
!bugfix

**Review by NonQueueingMatt — APPROVED** [review]
hallunicationary

**Comment by Llywelwyn** [comment]
> hallunicationary

hallunicationary

File: html/changelogs/llywelwyn-typos.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixed a lot of typos."
```
Comment (listerla) [inline html/changelogs/llywelwyn-typos.yml:41]: spellcheck is right there....

Comment (Llywelwyn) [reply html/changelogs/llywelwyn-typos.yml:41]: oh you're way smarter than me thanks, forgot that existed. will fix when at my pc

**Review by alsoandanswer — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**